src/main/webapp/resources/styles/shop.css

```diff
@@ -0,0 +1,3 @@
+body {
+
```

Comment:
What is this?

Reply:
file for front end, but ended up not doing front end. its there if someone decides to add front end..